tests/macro/html-component-fail.stderr

```diff
@@ -30,107 +30,155 @@ error: this open tag has no corresponding close tag
 57 |     html! { <Child with props > };
    |             ^^^^^^^^^^^^^^^^^^^
 
-error: unexpected token
+error: too many refs set
   --> $DIR/html-component-fail.rs:58:38
    |
 58 |     html! { <Child with props ref=() ref=() /> };
    |                                      ^^^
 
-error: unexpected token
-  --> $DIR/html-component-fail.rs:59:27
+error: too many refs set
+  --> $DIR/html-component-fail.rs:59:38
+   |
+59 |     html! { <Child with props ref=() ref=() value=1 /> };
+   |                                      ^^^
+
+error: Using special syntax `with props` along with named prop is not allowed. This rule does not apply to special `ref` prop
+  --> $DIR/html-component-fail.rs:60:38
+   |
+60 |     html! { <Child with props ref=() value=1 ref=() /> };
+   |                                      ^^^^^
+
+error: Using special syntax `with props` along with named prop is not allowed. This rule does not apply to special `ref` prop
+  --> $DIR/html-component-fail.rs:61:31
+   |
+61 |     html! { <Child with props value=1 ref=()  ref=() /> };
+   |                               ^^^^^
+
+error: Using special syntax `with props` along with named prop is not allowed. This rule does not apply to special `ref` prop
+  --> $DIR/html-component-fail.rs:62:28
+   |
+62 |     html! { <Child value=1 with props  ref=()  ref=() /> };
+   |                            ^^^^
+
+error: Using special syntax `with props` along with named prop is not allowed. This rule does not apply to special `ref` prop
+  --> $DIR/html-component-fail.rs:63:35
+   |
+63 |     html! { <Child value=1 ref=() with props ref=() /> };
+   |                                   ^^^^
+
+error: Using special syntax `with props` along with named prop is not allowed. This rule does not apply to special `ref` prop
+  --> $DIR/html-component-fail.rs:64:43
    |
-59 |     html! { <Child ref=() with props /> };
+64 |     html! { <Child ref=() ref=() value=1  with props  /> };
+   |                                           ^^^^
+
+error: Using special syntax `with props` along with named prop is not allowed. This rule does not apply to special `ref` prop
+  --> $DIR/html-component-fail.rs:65:27
+   |
+65 |     html! { <Child ref=() with props /> };
```

Comment:
This test doesn't look right

Reply:
@jstarry It is looks like currently on master if you use this approach, you will receive an error

Reply:
Yes, currently on master this will cause an error. My concern is that the test error message is confusing for this test case: 

```
"error: Using special syntax `with props` along with named prop is not allowed. This rule does not apply to special `ref` prop"
```